packages/grafana-ui/src/components/PanelChrome/PanelChrome.tsx

```diff
@@ -124,6 +125,18 @@ export function PanelChrome({
           </h6>
         )}
 
+        {panelNotices && typeof panelNotices.getPanelNotices === 'function' && (
```

Comment:
Think the `typeof panelNotices.getPanelNotices === 'function'` is not necessary here since it cannot be anything else?